src/prefect/tasks/aws/batch.py

```diff
@@ -91,7 +91,7 @@ def run(
                 **batch_kwargs,
             )
         except Exception as e:
-            raise FAIL(f"Failed to submit job '{job_name}' to AWS Batch.") from e
+            raise FAIL(f"Failed to submit job '{job_name}' to AWS Batch with exception: {e}.") from e
```

Comment:
Thanks for contributing!

Users can't opt out of state messages like this so this may leak information to Cloud that they do not want to share. This message will also just stringify the exception which loses a lot of information. Generally, it's better to use the `exc_info` feature of the logger which will include the traceback and such. If users don't want to share this information with Cloud, it's easily toggled with the flow run logging flag.

```suggestion
            self.logger.error("Failed to submit job", exc_info=True)
            raise FAIL(f"Failed to submit job '{job_name}' to AWS Batch.") from e
```

Reply:
got it, thanks!

Reply:
If you want to commit this change and give it a try that should be it!